mkdocs-community.yml

```
@@ -78,6 +83,8 @@ nav:
     - General technical: training/general_technical.md
     - Modelling: training/modelling.md
     - Analysis: training/analysis.md
+  - Events:
+    - events/events.md
```

```suggestion
    - events/index.md
```
The index pages need to be named `index.md`. Put a level 1 Header to put a nice title, e.g Events.

---

Thanks!